react-native-sdk/package.json

```diff
@@ -11,83 +11,83 @@
         "url": "git+https://github.com/jitsi/jitsi-meet.git"
     },
     "dependencies": {
-        "@jitsi/js-utils": "0.0.0",
-        "@jitsi/logger": "0.0.0",
-        "@jitsi/rtcstats": "0.0.0",
-        "@react-navigation/bottom-tabs": "0.0.0",
-        "@react-navigation/elements": "0.0.0",
-        "@react-navigation/material-top-tabs": "0.0.0",
-        "@react-navigation/native": "0.0.0",
-        "@react-navigation/stack": "0.0.0",
-        "@xmldom/xmldom": "0.0.0",
-        "base64-js": "0.0.0",
-        "grapheme-splitter": "0.0.0",
-        "i18n-iso-countries": "0.0.0",
-        "i18next": "0.0.0",
-        "js-md5": "0.0.0",
-        "i18next-http-backend": "0.0.0",
-        "js-sha512": "0.0.0",
-        "jwt-decode": "0.0.0",
-        "lib-jitsi-meet": "0.0.0",
-        "lodash": "0.0.0",
-        "moment": "0.0.0",
-        "moment-duration-format": "0.0.0",
-        "optional-require": "0.0.0",
-        "promise.allsettled": "0.0.0",
-        "punycode": "0.0.0",
-        "react-emoji-render": "0.0.0",
-        "react-i18next": "0.0.0",
-        "react-linkify": "0.0.0",
-        "react-native-dialog": "0.0.0",
-        "react-native-svg-transformer": "0.0.0",
-        "react-native-tab-view": "0.0.0",
-        "react-native-url-polyfill": "0.0.0",
-        "react-native-youtube-iframe": "0.0.0",
-        "react-redux": "0.0.0",
-        "redux": "0.0.0",
-        "redux-thunk": "0.0.0",
-        "unorm": "0.0.0",
-        "util": "0.0.0",
-        "uuid": "0.0.0",
-        "zxcvbn": "0.0.0"
+        "@jitsi/js-utils": "2.2.1",
```

Comment:
Please do not commit changes to dependency versions in react-native-sdk/package.json, keep the versions at 0.0.0. I know the dependency versions are updated whenever running `npm pack` so commit all changes before running that command

Reply:
ok